en/components/grid-summaries.md

```diff
@@ -34,15 +32,6 @@ The Grid Summary Cell comes with basic styling flexibility through the various o
 
 <img class="responsive-img" src="../images/grid_summaries_styling.png" srcset="../images/grid_summaries_styling@2x.png 2x" />
```

Comment:
@YoannaIvanova Make a new image where both available and unavailable summary cells have the same background color. The current image looks broken.